packages/plugins/posthog-ui/src/lib/services/posthog.service.ts

```
@@ -0,0 +1,476 @@
+ * - Error monitoring
+ */
+@Injectable({
+	providedIn: 'root'
+})
+export class PostHogService {
+	private initialized = false;
+	private config: any = {};
+
+	constructor(
+		private router: Router,
+		@Inject(PLATFORM_ID) private platformId: Object,
```

Biome (1.9.4)

[error] Don't use 'Object' as a type.

Prefer explicitly define the object shape. This type means "any non-nullable value", which is slightly better than 'unknown', but it's still a broad type.

(lint/complexity/noBannedTypes)